modules/tinymce/src/themes/silver/main/ts/ReadOnly.ts

```diff
@@ -75,17 +64,22 @@ export const setupReadonlyModeSwitch = (editor: Editor, uiComponents: RenderUiCo
   }
 };
 
-export const createReadonlyReceivingForOverflow = (getOverflow: (comp: AlloyComponent) => Option<AlloyComponent>) => {
+const receivingConfig = (): Behaviour.NamedConfiguredBehaviour<any, any> => {
   return Receiving.config({
     channels: {
       [ReadOnlyChannel]: {
         schema: ReadOnlyDataSchema,
-        onReceive: (comp, data: ReadOnlyData) => {
-          getOverflow(comp).each((toolbar) => {
-            setDisabledAll(toolbar, data.readonly);
-          });
+        onReceive(comp, data: ReadOnlyData) {
+          Disabling.set(comp, data.readonly);
         }
       }
     }
   });
 };
+
+export default {
```

Comment:
Default exports are an antipattern and prevent tree shaking, so we shouldn't be using them outside public APIs.

- [ ] Remove default export